### PR TITLE
jest-styled-components

### PR DESCRIPTION
**Pull request by mortigen**

### CODE REVIEW

File: template/src/.storybook/storyshots.test.js [hunk 1]
```diff
@@ -1,22 +1,22 @@
...
+      snapshotsDirName: './__tests__/__snapshots__',
+    });
+    const snapshotFilename = converter.getSnapshotFileName(context);
+    const storyElement = story.render(context);
 
-  stories2snapsConverter: new Stories2SnapsConverter({
-    snapshotsDirName: './__tests__/__snapshots__',
-  }),
+    const {container} = render(storyElement)
+    expect(container.firstChild).toMatchSpecificSnapshot(snapshotFilename);
```
Comment (mikeqcp) [inline template/src/.storybook/storyshots.test.js]: comparing firstChild assumes the component will render only one child (which is not true if it returns fragment or array of elements) - don't you think we should use only `container`?

File: template/src/routes/__tests__/__snapshots__/app.component.spec.js.snap [hunk 2]
```diff
@@ -1,11 +0,0 @@
-// Jest Snapshot v1, https://goo.gl/fbAQLP
```
Comment (mikeqcp) [inline template/src/routes/__tests__/__snapshots__/app.component.spec.js.snap]: Why are all these snapshots removed?

Comment (mortigen) [reply template/src/routes/__tests__/__snapshots__/app.component.spec.js.snap]: I thought we want to remove all snapshots from tamplate. It's not necessary to keep them here but if you want then I will get them back

Comment (mikeqcp) [reply template/src/routes/__tests__/__snapshots__/app.component.spec.js.snap]: I would keep them so if you install the project and run `yarn test` all tests should pass by defaulut (including snapshots)

**Review by mikeqcp — APPROVED** [review]

**Pull request merged**